src/SlashCommands.tsx

```diff
@@ -768,18 +771,50 @@ export const Commands = [
     }),
     new Command({
         command: 'addwidget',
-        args: '<url>',
+        args: '<url | embed code | jitsi url>',
```

Comment:
Maybe uppercase Jitsi here?

```suggestion
        args: '<url | embed code | Jitsi url>',
```